coucharchive

```diff
@@ -416,10 +419,10 @@ def load(filename):
     _load_archive(filename, callback)
 
 
-def restore(target, filename, ignore_dbs=[]):
+def restore(target, filename, force_continue, ignore_dbs=[]):
```

Comment:
Since it's an optional option on the command line, it would feel more natural with an optional argument here also:
```
force_continue=False,
```